newton/_src/solvers/mujoco/solver_mujoco.py

```
@@ -1719,9 +1712,30 @@ def add_geoms(newton_body_id: int):
+            eq = spec.add_equality(objtype=mujoco.mjtObj.mjOBJ_BODY)
+            eq.type = mujoco.mjtEq.mjEQ_CONNECT
+            eq.active = True
+            eq.name1 = model.body_key[joint_parent[j]]
+            eq.name2 = model.body_key[joint_child[j]]
+            eq.data[0:3] = joint_child_xform[j][:3]
+            mjc_eq_to_newton_jnt[eq.id] = j
+
+        if skip_visual_only_geoms and len(spec.geoms) != colliding_shapes_per_world:
+            raise ValueError(
+                "The number of geoms in the MuJoCo model does not match the number of colliding shapes in the Newton model."
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)